Common/Market.cs

```diff
@@ -118,6 +119,11 @@ static Market()
         /// GDAX
         /// </summary>
         public const string GDAX = "gdax";
+        
+        /// <summary>
+        /// AU
+        /// </summary>
```

Comment:
nit - perhaps spell out Australia (maybe not everyone knows AU == Australia?)

Reply:
I was instructed by @jaredbroad to use AU over Aus or Australia after some discussion in our slack channel. That said, this is commented out so perhaps we could change it to Australia for clarity?

Reply:
Agreed with using `Market.AU` -- matches `Market.USA` -- comment was only in regards to the xml comment on line 124.

Looks good as updated, thanks.